app/src/main/java/blockeq/com/stellarwallet/activities/ShowMnemonicActivity.kt

```diff
@@ -103,7 +103,7 @@ class ShowMnemonicActivity : BaseActivity(), View.OnClickListener {
             }
 
             mnemonicString = String(mnemonic)
-            return String(mnemonic).split(" ".toRegex()).dropLastWhile { it.isEmpty() } as ArrayList
+            return mnemonicString!!.split(" ".toRegex()).dropLastWhile { it.isEmpty() } as ArrayList
```

Comment:
why did you put !!? it looks like `mnemonicString` can not be null since is assigned to a new String object

Reply:
Kotlin's compiler apparently is not perfect and they don't allow it :(